importer/BeBruNl.py

```diff
@@ -0,0 +1,131 @@
+                         for x in types if x.lower() == objtype]
+        if len(special_types) > 0:
+            self.remove_statement("is")
+            for q in special_types[0]:
+                self.add_statement("is", q)
+
+    def set_style(self):
+        if self.has_non_empty_attribute("bouwstijl"):
+            style_raw = self.bouwstijl.lower()
+            styles = self.data_files["style"]["mappings"]
+            arch_style = [styles[x]["items"]
+                          for x in styles if x.lower() == style_raw]
+            if len(arch_style) > 0:
+                for q in arch_style[0]:
+                    self.add_statement("architectural_style", q)
+            else:
+                self.add_to_report(
+                    "bouwstijl", self.bouwstijl, "architectural_style")
+
+    def set_architect(self):
+        if self.has_non_empty_attribute("bouwdoor"):
+            if utils.count_wikilinks(self.bouwdoor) == 1:
+                arch_q = utils.q_from_first_wikilink("nl", self.bouwdoor)
+                self.add_statement("architect", arch_q)
+            else:
+                self.add_to_report("bouwdoor", self.bouwdoor, "architect")
+
+    def update_labels(self):
+        dutch = utils.remove_markup(self.omschrijving)
+        self.add_label("nl", dutch)
+
+    def update_descriptions(self):
+        dutch = "{} in {}, België".format(self.objtype.lower(), self.plaats)
+        self.add_description("nl", dutch)
+
+        english = "heritage site in {}, Belgium".format(self.plaats)
+        self.add_description("en", english)
+
+    def set_building_year(self):
+        if self.has_non_empty_attribute("bouwjaar"):
+            if utils.legit_year(self.bouwjaar):
+                self.add_statement(
+                    "inception", {"time_value": {"year": self.bouwjaar}})
+            else:
+                self.add_to_report("bouwjaar", self.bouwjaar, "inception")
+
+    def set_monuments_all_id(self):
+        self.monuments_all_id = self.code
+
+    def set_heritage_id(self):
+        self.add_statement("heritage_brussels", self.code)
+
+    def exists_with_monument_article(self, language):
+        return super().exists_with_monument_article("nl", "monument_article")
+
+    def __init__(self, db_row_dict, mapping, data_files, existing, repository):
+        Monument.__init__(self, db_row_dict, mapping,
+                          data_files, existing, repository)
+        self.set_monuments_all_id()
+        self.set_changed()
+        self.wlm_source = self.create_wlm_source(self.monuments_all_id)
+        self.set_source()
+        self.set_registrant_url()
+        self.set_heritage_id()
+        self.set_heritage()
+        self.set_country()
+        self.set_image()
+        self.set_adm_location()
+        self.set_commonscat()
+        self.set_architect()
+        self.set_style()
+        self.set_is()
+        self.set_special_is()
+        self.set_coords(("lat", "lon"))
```

Comment:
I would recomment modifying `Monument.set_coords()` to allow `coord_keywords_tuple` to be `None` then add `coord_keywords_tuple = coord_keywords_tuple or ("lat", "lon")` inside it since that is the dominating tuple (used for all newer mappings).

Then the call here would just be `self.set_coords()`

Reply:
Done.